src/test/java/com/hubspot/jinjava/lib/tag/IncludeTagTest.java

```diff
@@ -101,4 +101,13 @@ public String getString(String fullName, Charset encoding,
     assertThat(result.getOutput().trim()).isEqualTo("INCLUDED");
   }
 
+  @Test
+  public void itSetsErrorLineNumbersCorrectly() throws IOException {
+
+    RenderResult result = jinjava.renderForResult(Resources.toString(Resources.getResource("tags/includetag/errors/base.html"), StandardCharsets.UTF_8),
+        new HashMap<>());
+
+    assertThat(result.getErrors()).hasSize(1);
+    assertThat(result.getErrors().get(0).getLineno()).isEqualTo(7);
```

Comment:
Previously this would be on line number 4 of `base.html` highlighting `{% set test = 1 + 1 %}` as the error rather than `{{ "test"|add(1) }}` in the included file.